tests/Jellyfin.Naming.Tests/Video/CleanDateTimeTests.cs

```diff
@@ -1,143 +1,59 @@
-
-            Test(@"/server/Movies/300 (2007)/300 (2006)", "300", 2006);
-            Test(@"/server/Movies/300 (2007)/300 (2006).mkv", "300", 2006);
-        }
-
-        [Fact]
-        public void TestCleanDateTimeWithoutDate()
-        {
-            Test(@"American.Psycho.mkv", "American.Psycho.mkv", null);
-            Test(@"American Psycho.mkv", "American Psycho.mkv", null);
-        }
-
-        [Fact]
-        public void TestCleanDateTimeWithBracketedName()
-        {
-            Test(@"[rec].mkv", "[rec].mkv", null);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestCleanDateTimeWithoutExtension()
-        {
-            Test(@"St. Vincent (2014)", "St. Vincent", 2014);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestCleanDateTimeWithoutDate1()
-        {
-            Test("Super movie(2009).mp4", "Super movie", 2009);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestCleanDateTimeWithoutParenthesis()
-        {
-            Test("Drug War 2013.mp4", "Drug War", 2013);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestCleanDateTimeWithMultipleYears()
-        {
-            Test("My Movie (1997) - GreatestReleaseGroup 2019.mp4", "My Movie", 1997);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestCleanDateTimeWithYearAndResolution()
-        {
-            Test("First Man 2018 1080p.mkv", "First Man", 2018);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestCleanDateTimeWithYearAndResolution1()
-        {
-            Test("First Man (2018) 1080p.mkv", "First Man", 2018);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestCleanDateTimeWithSceneRelease()
-        {
-            Test("Maximum Ride - 2016 - WEBDL-1080p - x264 AC3.mkv", "Maximum Ride", 2016);
-        }
-
-        // FIXME
-        // [Fact]
-        public void TestYearInBrackets()
-        {
-            Test("Robin Hood [Multi-Subs] [2018].mkv", "Robin Hood", 2018);
-        }
-
-        private void Test(string input, string expectedName, int? expectedYear)
+        private readonly NamingOptions _namingOptions = new NamingOptions();
+
+        [Theory]
+        [InlineData(@"The Wolf of Wall Street (2013).mkv", "The Wolf of Wall Street", 2013)]
+        [InlineData(@"The Wolf of Wall Street 2 (2013).mkv", "The Wolf of Wall Street 2", 2013)]
+        [InlineData(@"The Wolf of Wall Street - 2 (2013).mkv", "The Wolf of Wall Street - 2", 2013)]
+        [InlineData(@"The Wolf of Wall Street 2001 (2013).mkv", "The Wolf of Wall Street 2001", 2013)]
+        [InlineData(@"300 (2006).mkv", "300", 2006)]
+        [InlineData(@"d:/movies/300 (2006).mkv", "300", 2006)]
+        [InlineData(@"300 2 (2006).mkv", "300 2", 2006)]
+        [InlineData(@"300 - 2 (2006).mkv", "300 - 2", 2006)]
+        [InlineData(@"300 2001 (2006).mkv", "300 2001", 2006)]
```

Comment:
I kind of liked the method names explaining the individual tests for these files.